assets/js/src/form_editor/store/reducers/history_record.ts

```diff
@@ -0,0 +1,83 @@
+import { isEqual } from 'lodash';
+
+export const createHistoryRecord = (state) => {
+  let editorHistory: any[] = state.editorHistory;
```

Comment:
It is great that you are using types here. But please don't use `any`, in my branch I updated TS to disallow this. The build would fail.
You know the structure, right? So it should be easier to create types for this.

Reply:
Right, it's a nice place for type.